synapse/rest/client/versions.py

```diff
@@ -96,7 +96,7 @@ def on_GET(self, request: Request) -> Tuple[int, JsonDict]:
                     "io.element.e2ee_forced.private": self.e2ee_forced_private,
                     "io.element.e2ee_forced.trusted_private": self.e2ee_forced_trusted_private,
                     # Supports the busy presence state described in MSC3026.
-                    "org.matrix.msc3026.busy_presence": self.config.experimental.msc3026_enabled,
+                    "org.matrix.msc3026.busy_presence": True,
```

Comment:
I don't think this is really accurate -- don't we need to check this based on which user is requesting `/versions`?

For example, a client might use this to enable UI allowing users to say they're busy. The user would then receive an unexpected error when attempting to use that feature.

(Similar feedback to below for MSC3881)

Reply:
Hmm I think you're right - it would be True or False based on either the config or whether the feature is enabled in the DB, but how would that be checked here per user? Is it possible to extract the username from the `GET` request to `/versions` (my suspicion is no?)

Reply:
Nevermind I figured it out.

Reply:
Well I thought I figured it out but since `/versions` doesn't require authentication there's no access token to extract the user id from. Is there a way to figure out the user id from a request that doesn't require authentication? If not, what's the best way to handle what `/versions` should return here if we can't determine the user (and thus cannot check if it is enabled per user)?

Reply:
The only thing I can think of is to make it optionally require authentication and ask the clients to pretty please change their request code so it sends auth for `/versions`... :/

Reply:
> Hmm, I wonder if we should start putting unstable features in `/capabilities` API (which _is_ authed) instead?

I think at some point I was told to prefer `/versions` over `/capabilities`, but after some searching I can't seem to find a reference to this.

Reply:
[MSC4026](https://github.com/matrix-org/matrix-spec-proposals/pull/4026) was written for this, it has now passed FCP.